configs/components/oifs/oifs-43r3-preprocess.sh

```diff
@@ -52,6 +52,7 @@ elif [[ "$(hostname -f)" =~ juwels ]] ; then
     module load ParaStationMPI/5.4.4-1-mt    
     module load Python/3.6.8
     module load imkl/2019.3.199
+    export PATH=$IO_LIB_ROOT/bin:$PATH:/p/project/chhb19/jstreffi/software/cdo/cdo-1.9.8/src/
```

Comment:
We can keep this for now, but the restrictive folder access policy of the JSC means that this will not work for users that are not part of project chhb19. Unfortunately JSC has been uncooperative in installing software packages system-wide for us. There is no better solution at the moment. AFAIK the CDO in the latest stages does not work as we need it.

Reply:
Maybe we can add a comment that clarifies to future users, that the may have to install their own cdo and change this path to point towards that installation.

Reply:
Alternatively all paths that are not accessible to all users on Juwels can go into the exp definition yaml. 
In fact I like that idea a lot more, since it put the variable that likely needs to be changed at the top user interface level!